docs/quick-start.rst

```diff
@@ -28,7 +28,7 @@ Getting started
 
 1. Get SNARE: ``git clone https://github.com/mushorg/snare.git`` and ``cd snare``
 
-2. Install requirements: ``pip3 install -r requirements.txt``
+2. Install requirements: ``sudo pip3 install -r requirements.txt``
```

Comment:
Do we really want to install libraries with sudo?

Reply:
Hi @afeena 
Yes, actually the libraries are to be installed in the global pip environment because while running the subsequent commands it might result in an error in some cases.
You can verify this from the README.md as well. It has also been changed accordingly. Also check #178